src/app/pages/storage/disks/disk-list/disk-list.component.ts

```diff
@@ -342,7 +342,7 @@ export class DiskListComponent implements EntityTableConfig<Disk> {
           item.type = entityDialog.formValue.type;
         });
 
-        this.ws.call('smart.test.manual_test', [disksIdentifier]).pipe(untilDestroyed(parent)).subscribe(
+        this.ws.call('smart.test.manual_test', [disksIdentifier]).pipe(untilDestroyed(parent)).subscribe( // eslint-disable-line no-restricted-globals
```

Comment:
This is likely a leftover from refactoring. Likely should be `untilDestroyed(this)`. Needs testing.